automatic/wps-office-free/wps-office-free.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>wps-office-free</id>
+    <version>10.1.0.5820</version>
+    <title>WPS Office Free</title>
+    <owners>chocolatey</owners>
+    <authors>Kingsoft Software</authors>
+    <projectUrl>http://www.ksosoft.com/product/office-free</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/e4a49519947c3cff55c17a0b08266c56b0613e64/icons/kingsoft-office-free.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    Windows WPS Office Free The perfect free office software Your dream solution for a low-budget, no-commitment, free productivity office suite. The smallest size office suite consisting of Writer, Presentation, and Spreadsheets, plus PDF tools to fulfill all of your document needs. Includes a brand new Docer Templates page. Available in English, French, German, Spanish, Portuguese, Polish and Russian. This Package only installs the English Language version.
```

Comment:
Consider adding a `##Features`` section, with the content from http://www.ksosoft.com/product/office-free (Key Features)